core/mondoo-dns-security.mql.yaml

```
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: dns
```

would this be `host, dns` 🤔 

```suggestion
      mondoo.com/platform: host, dns
```

---

I think host does not apply here